pipelines/main/misc/upload_buildkite_results.yml

```
@@ -0,0 +1,30 @@
+steps:
+  - group: "Upload"
+    steps:
+      - label: "upload test results ${TRIPLET?}"
+        key:   "upload_results_${TRIPLET?}"
+        depends_on:
+          # Wait for the builder to finish
+          - step: "build_${TRIPLET?}"
+          # Wait for the tester to finish
+          - step: "test_${TRIPLET?}"
+            allow_failure: true
+        plugins:
+          - staticfloat/cryptic#v2:
+              variables:
+                - BUILDKITE_ANALYTICS_TOKEN="U2FsdGVkX18BUSpNWNimRUOAh2ku0ugS7x3RRy1CfJBCxTLWArJpa1PXKoILfI7q"
+          - test-collector#v1.10.1:
+              files: "results.json"
+              format: "json"
+        timeout_in_minutes: ${TIMEOUT?}
+        soft_fail: true
+        commands: |
+          buildkite-agent artifact download --step "test_${TRIPLET?}" "*/results.json" .
```

```suggestion
          buildkite-agent artifact download --step "test_${TRIPLET?}" "results.json" .
```

We will see in a bit, but I think now the `/` will mess us up.